util/libcrypto.num

```diff
@@ -1808,6 +1808,7 @@ CRYPTO_THREAD_init_local                1850	3_0_0	EXIST::FUNCTION:
 TS_RESP_CTX_set_serial_cb               1851	3_0_0	EXIST::FUNCTION:TS
 POLICY_MAPPING_it                       1852	3_0_0	EXIST::FUNCTION:
 ERR_load_KDF_strings                    1853	3_0_0	EXIST::FUNCTION:DEPRECATEDIN_3_0
+ERR_load_KDF_strings                    1853	3_0_0	EXIST::FUNCTION:DEPRECATEDIN_3_0
```

Comment:
We seem to have ERR_load_KDF_strings twice here???

Reply:
travis does not like this :)

Reply:
Uhmmmm, I have no idea how this happened.  I'll simply remove one and see if that helps...  this might just be a rebase thing (I have moved around stuff a bit gone rogue

Reply:
Duplicate declarations was the cause.  Fixed

Reply:
I don't get it.  Doing a full rebuild, make update, etc., takes a few minutes (much less if you use ccache) also thanks to the parallel test runs now. And it avoids almost all build surprises which mean another needless submit/CI cycle.  It's worth it.